.babelrc

```diff
@@ -1,14 +1,43 @@
 {
   "env": {
     "test": {
+      "presets": [
+        ["env", {"targets": { "node": "current" }}],
+        "react",
+        "stage-2"
+      ],
       "plugins": ["babel-plugin-rewire"]
     },
+    "development": {
+      "presets": [
+        ["env", { "targets": { "browsers": ["last 2 versions", "ie >= 9"] } }],
+        "react",
+        "stage-2"
```

Comment:
https://github.com/babel/babel-preset-env#support-all-plugins-in-babel-that-are-considered-latest